azure-go-sdk-conceptual/azure-sdk-go-authorization.md

```diff
@@ -141,4 +144,4 @@ certificateAuthorizer := auth.NewClientCertificateConfig(applicationID, certific
         certificatePassword, tenantID,
         azure.PublicCloud.ActiveDirectoryEndpoint, azure.PublicCloud.ResourceManagerEndpoint)

```

Comment:
also please note that the parameters here are incorrect https://github.com/Azure/go-autorest/blob/90128b0cbed977972cdf104474875758e9fe97d0/autorest/azure/auth/auth.go#L215